### DISCUSSION TITLE
13th release: Improving OpenTherm parsing

### DISCUSSION THREAD

**Discussion by rvdbreemen** [rvdbreemen/OTGW-firmware#57] [General]
This release improves the OpenTherm parsing and some more minor issues.

Improvements and fixes:
- Adding missing OT msgids (Issue #5) 
- Improved the Ventilation and Heating Recovering parsing
- Adding more MQTT messages due to improvements to parsing (more status bits are parsed)
- Adding PS=1 (print state) detection (WebUI stalls when PS=1 message is sent, eg. classic integrations Domoticz and HA)
- Bugfix to save the setting for eval interval setting
- Added more MQTT Home Assistant sensors (eg. status bits for Ventilation/Heating recovery, Solar Storage)
- Some breaking changes due to MQTT topic changes.

In short, a minor release, with no major feature. A lot of small bits added, improved or fixed. 

<hr /><em>This discussion was created from the release <a href='https://github.com/rvdbreemen/OTGW-firmware/releases/tag/v0.8.4'>13th release: Improving OpenTherm parsing</a>.</em>
